packages/@expo/cli/src/start/server/DevServerManager.ts

```diff
@@ -197,7 +197,7 @@ export class DevServerManager {
     if (!typescriptPrerequisite) {
       server.waitForTypeScriptAsync().then(async (success) => {
         if (success) {
-          await server.startTypeScriptServices();
+          server.startTypeScriptServices();
```

Comment:
This looks strange, but the await never did anything as `startTypeScriptServices` never returned a value.

The dev server shouldn't wait for function to complete, so it correct to remove the `await`.

Reply:
Should add a comment inline about this.